cypress/integration/resources.spec.js

```diff
@@ -155,49 +141,43 @@ describe('resources', () => {
   });
 
   it('will allow a user to filter resources by cost', () => {
-    cy.get(CostSelect).click({ force: true }).type('Paid').type('{enter}');
+    cy.get(CostSelect).click({ force: true }).type('Free').type('{enter}');
     cy.findByTestId(RESOURCE_SEARCH_BUTTON).click();
     cy.location().should(loc => {
       expect(loc.pathname).to.eq('/resources/1');
-      expect(loc.search).to.eq('?paid=true');
+      expect(loc.search).to.eq('?paid=false');
     });
 
-    cy.wait(1000);
-
-    cy.get('[data-test-paid]').each(card => {
-      cy.wrap(card).invoke('attr', 'data-test-paid').should('eq', 'true');
+    cy.findAllByTestId(RESOURCE_CARD).each(card => {
+      cy.wrap(card)
+        .get('[data-test-isPaid]')
+        .invoke('attr', 'data-test-isPaid')
```

Comment:
What's going on here with `.invoke()`?

I bet we can create a custom Cypress function that let's us check the value of a specific `data-test-*` attribute

https://docs.cypress.io/api/cypress-api/custom-commands.html

`cy.checkCustomDataAttribute('isPaid', false)` would be the usage perhaps?

Reply:
as far as `invoke()`, from docs:
`cy.get() yields a jQuery object, you can get its attribute by invoking the .attr() method.` 

changed to use different chainers:
`cy.get('[data-test-id="test-example"]')
  .should('have.attr', 'data-test-id', 'test-example')`
 in a Cypress function `checkCustomDataAttribute` as you had suggested